addons/crm/crm_lead.py

```diff
@@ -730,28 +730,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, section_
 
         return True
 
+    @api.one
```

Comment:
You can't put @api.one due to:

* This is old API. You're mixing both APIs in the same file. I don't know what is the Odoo policy about this.
* api.one encapsulates the result in a list, so the unwrapping is not always correctly done.